### PR TITLE
README.md for automateEC2.sh 

### PR DESCRIPTION
**Pull request by rcarterjr**
Here is the readme to run the script.

### CODE REVIEW

File: test/automation/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Automate EC2 Cluster
+
+The automateEC2 script is an executable that allows you to set up Accumulo on an AWS cluster for testing purposes. The cluster_props file is where all of the properties are defined for the script.
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
The `automateEC2` script is an executable that allows you to set up Accumulo on an AWS cluster for testing purposes. The `cluster_props` file is where all of the properties are defined for the script.
```

File: README.md [hunk 2]
```diff
@@ -238,6 +238,9 @@ test and produce json result files.
 There are some utilities for working with the json result files, run the `performance` script
 with no options to see them.
 
+## Run Accumulo on EC2 Test Cluster
+see the [readme.md](/test/automation/README.md)
```
Comment (ctubbsii) [inline README.md]:
```suggestion
See the [readme.md](/test/automation/README.md).
```

File: test/automation/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+All repositories are set to the master branch of the apache projects by default. You can change these values to your specific forks and branches as desired.
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
All repositories are set to the master branch of the corresponding Apache project by default. You can change these values to your specific forks and branches as desired.
```

File: test/automation/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+A path to muchos.props is required in order to run the script. You can find the required configurations and an example of muchos.props [in the official Fluo-Muchos readme](https://github.com/apache/fluo-muchos).
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
A path to `muchos.props` is required in order to run the script. You can find the required configurations and an example of `muchos.props` [in the official Fluo-Muchos readme](https://github.com/apache/fluo-muchos).
```

File: test/automation/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+### Execution takes no arguments as followed: 
+	./automateEC2.sh
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
    ./automateEC2.sh
```

File: test/automation/README.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+### Things to consider:
+* The script will read from the MUCHOS_PROPS environment variable if defined. 
+
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
```

File: test/automation/README.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+* Accumulo versions between muchos.props and pom file of a given Accumulo branch should be consistent.
+
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
```

File: test/automation/README.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+All repositories are set to the master branch of the apache projects by default. You can change these values to your specific forks and branches as desired.
+
+A path to muchos.props is required in order to run the script. You can find the required configurations and an example of muchos.props [in the official Fluo-Muchos readme](https://github.com/apache/fluo-muchos).
+
+### Execution takes no arguments as followed: 
+	./automateEC2.sh
+
+### Things to consider:
+* The script will read from the MUCHOS_PROPS environment variable if defined.
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
* The script will read from the `MUCHOS_PROPS` environment variable if defined. 
```

File: test/automation/README.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+A path to muchos.props is required in order to run the script. You can find the required configurations and an example of muchos.props [in the official Fluo-Muchos readme](https://github.com/apache/fluo-muchos).
+
+### Execution takes no arguments as followed: 
+	./automateEC2.sh
+
+### Things to consider:
+* The script will read from the MUCHOS_PROPS environment variable if defined. 
+
+* Accumulo versions between muchos.props and pom file of a given Accumulo branch should be consistent.
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
* Accumulo versions between `muchos.props` and `pom.xml` file of a given Accumulo branch should be consistent.
```

File: test/automation/README.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+* Accumulo and Fluo-Muchos will be installed locally in a temporary directory under /tmp which is typically removed every reboot.
```
Comment (ctubbsii) [inline test/automation/README.md]:
```suggestion
* Accumulo and Fluo-Muchos will be installed locally in a temporary directory under `/tmp`, which is typically cleared by the operating system periodically (system-specific, possibly on reboots, daily, weekly, etc.).
```

**Comment by keith-turner** [comment]
@rcarterjr I was attempting to create a PR through github's website and it glitched out and created a commit on your branch.  I hope this does not cause you any problems.

File: README.md [hunk 11]
```diff
@@ -238,6 +238,9 @@ test and produce json result files.
 There are some utilities for working with the json result files, run the `performance` script
 with no options to see them.
 
+## Run Accumulo on EC2 Test Cluster
```
Comment (keith-turner) [inline README.md]: I think this title is misleading, running Accumulo on EC2 is what Muchos does.  The script automates setting up a snapshot version of Accumulo and Accumulo testing on a cluster.  I am not sure what a better title would be, that is also short.  Maybe `Automating cluster testing` would be a good title.

Comment (rcarterjr) [reply README.md]: OK, I just changed the front page readme to be consistent with your change.

**Pull request merged**